src/Persistance/RepositoryImplementations/SystemRegisterRepository.cs

```diff
@@ -230,19 +263,19 @@ FROM business_application.system_register
     /// <inheritdoc/> 
     public async Task<bool> UpdateRightsForRegisteredSystem(List<Right> rights, string systemId)
     {
-        const string QUERY = /*strpsql*/@"
+        const string QUERY = /*strpsql*/"""            
```

Comment:
We have to also handle the update of clientid here, in case there is any new client id. If the request removes an existing client id, we need to agree with the product owner on how we handle it. Do we need to maintain the history by setting it to soft delete or they can be hard deleted.

Reply:
will be handled in a new PR